### PR TITLE
Add ability to delete workouts

### PR DESCRIPTION
**Pull request by dremex**
Adds a delete button next to the existing edit button for workouts. This is helpful when a user would like to completely remove a workout.

Based upon a request from twitter (https://twitter.com/TheGrantDesign/status/660702939946729472) and my own needs. :)

### CODE REVIEW

File: past-workouts.php [hunk 1]
```diff
@@ -203,8 +211,26 @@ function convertClimbTabletoWorkoutArray($climbTable) {
...
 			};
+
+			$("#confirmWorkoutDeleteModal").on("shown.bs.modal", function(e) {
+				$(this).find('.btn-ok').data('workoutId', $(e.relatedTarget).data('workoutId'));
+			});
+
+			$('#confirmDeleteButton').click(function() {
+				var workoutId = $(this).data("workoutId");
+				$.ajax({
+					url: sitePath() + "/workout-delete?wid=" + workoutId,
```
Comment (shimizust-zz) [inline past-workouts.php]: This was showing an ajax error for me. Fixed by adding ".php" to the path: url: 

```
sitePath() + "/workout-delete.php?wid=" + workoutId,
```

File: workout-delete.php [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+$workoutLoggingService = new WorkoutLoggingService();
+
+$workoutInfo = $workoutLoggingService->getWorkoutInfo($workoutId);
+$workoutUserId = $workoutInfo["userid"];
+
+// Check that user of the workout_id_prev matches that of the cookie userid
+if ($userid != $workoutUserId) {
+    header('HTTP/1.1 500 Internal Server Error');
+} else {
+    $workoutLoggingService->deleteWorkout($workoutId);
```
Comment (shimizust-zz) [inline workout-delete.php:20]: After deleting the workout, we need to update the user records table, which may have changed upon deleting the workout. Just add the following after this line:

```
//update records table
include 'update-records-absolute.php';
```

**Comment by dremex** [comment]
Thanks for noticing the broken url, I must have removed that before checking in. Also thanks for the note about the user records, that one I totally missed.

**Comment by shimizust-zz** [comment]
Thanks for adding this useful feature. I've been wanting this as well!

**Pull request merged**